src/extension/excel/src/main/java/org/geoserver/wfs/response/ExcelOutputFormat.java

```diff
@@ -1,30 +1,17 @@
-/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
- * (c) 2001 - 2013 OpenPlans
```

Comment:
I don't think this should be removed

Reply:
For new files, use 2025. For existing files, don't touch the copyright notice.